string nodeName
---
int8 ret
bool res

